packages/default-storage/example/app.json

```diff
@@ -3,12 +3,14 @@
   "displayName": "AsyncStorageExample",
   "components": [
     {
-      "appKey": "AsyncStorageExample"
+      "appKey": "AsyncStorageExample",
+      "displayName": "AsyncStorageExample"
     }
   ],
   "resources": {
     "android": ["index.android.jsbundle"],
     "ios": ["index.ios.jsbundle"],
-    "macos": ["index.macos.jsbundle"]
+    "macos": ["index.macos.jsbundle"],
+    "windows": ["main.windows.bundle"]
```

Comment:
Is this right? Why is it the only platform using `main.*`?

Reply:
it's because of our building scripts outputting `index.*.jsbundle`. I changed it to `main.*.bundle` to make them all align